### DISCUSSION TITLE
user created using a default user

### DISCUSSION THREAD

**Discussion by dbz195** [hrfee/jfa-go#147] [Q&A]
user created using a default user, anyway to copy all profile setting not just home layout?

**Comment by hrfee** [comment]
Not quite sure what you mean, but the last time I checked, all settings that are possible to apply are already applied. Which settings aren't copying for you?

**Reply by dbz195** [reply]
exp: 
- Maximum number of simultaneous user sessions
- Allow media downloads
-  Allow remote control of other users
- Allow remote control of shared devices
- SyncPlay access
- TV access

**Reply by hrfee** [reply]
All of these settings should be set by jfa-go, could you try applying your profile to a user, then check the programs logs to see if there are any errors?
